src/main/java/kafka/TaskList.java

```diff
@@ -0,0 +1,51 @@
+package kafka;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    public final ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTask(Task task) {
+        this.tasks.add(task);
+    }
+
+    public void printList() {
+        if (tasks.isEmpty()) {
+            return;
+        }
+        String listMessage = "";
```

Comment:
Minor nitpick, but you could consider using StringBuilder for some improvement in time complexity. It is really minor though as the strings are probably not long enough to result in any real world performance benefits :)